salt/utils/schedule.py

```diff
@@ -713,7 +734,6 @@ def handle_func(self, multiprocessing_enabled, func, data):
 
                 ret['success'] = True
 
-                data_returner = data.get('returner', None)
```

Comment:
I'm not following the reasoning for this change. Help? :)

Reply:
@cachedout Definitely.  I moved it outside the try...except.  When an exception happens, eg. the new one I added for minion_blackout or if an invalid call happens, the code will drop into the `finally` section where data_returner is referenced.  In some situations it hasn't been created yet, so errors appear in the logs.

Reply:
Makes perfect sense. Thank you!